docs/bigquery-usage.rst

```diff
@@ -397,13 +386,17 @@ the job locally:
 .. doctest::
 
    >>> from gcloud import bigquery
+   >>> from gcloud.bigquery import SchemaField
    >>> client = bigquery.Client()
    >>> table = dataset.table(name='person_ages')
-   >>> job = table.load_from_storage(bucket_name='bucket-name',
-   ...                               object_name_glob='object-prefix*',
-   ...                               source_format='CSV',
-   ...                               skip_leading_rows=1,
-   ...                               write_disposition='truncate')
+   >>> table.schema = [
```

Comment:
Did you previously envision the schema would be implied from the CSV? Is that even possible? (AFAIK the BQ API will just consume the whole CSV, but maybe we could read 8 (or 64, or 512, etc.) bytes at a time from the object in Cloud Storage until we had all the headers? Or maybe we should leave the delimiter guessing game up to the BQ API.)